Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -187,21 +166,60 @@ gdjs.LightRuntimeObjectPixiRenderer.prototype.getRendererObject = function () {
 gdjs.LightRuntimeObjectPixiRenderer.prototype.ensureUpToDate = function () {
   if (this._object.isHidden()) return;
 
-  if (this._debugMode) this._updateDebugGraphics();
+  if (this._debugGraphics) this._updateDebugGraphics();
```

Comment:
@4ian somehow the debug graphics is broken, I tried to run the previous commits as well, but I'm unable to render PIXI.Graphics, there is no error, and there's no visual as well.

Reply:
Is `_debugGraphics` set to something? Add some breakpoint/various console.log to inspect the content of the object?

Reply:
Yes, I could see that `_debugGraphics` is a valid object, and I am getting console logs for all the updating functions 🤔 I thought it might have to do with the refactoring (which is basically nothing more than extracting the logic to define the pixi mesh, shouldn't have to do anything with the graphics), but the graphics isn't rendering without the refactor as well.

Reply:
Could it be because "if (this._object.isHidden()) return;"? Have you verified that `ensureUpToDate` is called?
I see that @bouh was able to make screenshot with the debug mode, so it was working at this time :)

Reply:
@4ian nvm I found the bug 😬, I had to trace back all the changes, it was due to our optimization of falling back to quads and the empty array returned by `_computeLightVertices()`. Also, we should be hot-reload ready now 🤞.